chart/flux/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 appVersion: "1.18.0"
-version: 1.2.0
+version: 1.2.1
```

Comment:
The chart is not automagically released, and the version is thus only bumped while a human makes the release. Otherwise we would be skipping versions.